python/README.md

```diff
@@ -0,0 +1,19 @@
+#Delta Lake
+Delta Lake is an open source storage layer that brings reliability to data lakes. Delta Lake provides ACID transactions, scalable metadata handling, and unifies streaming and batch data processing. Delta Lake runs on top of your existing data lake and is fully compatible with Apache Spark APIs.
+
+<https://delta.io>
+
+## Online Documentation
+
+You can find the latest Delta Lake documentation on the [project web page](https://docs.delta.io/latest/delta-intro.html)
+
+## Python Packaging
+
+This README file only contains basic information related to pip installed Delta Lake.
+This packaging is currently experimental and may change in future versions (although we will do our best to keep compatibility).
+
+## Python Requirements
+
+Pyspark >= 3.0.0
```

Comment:
do we need to write the requirements here. this is something that is going to change. and it will be a pain to update.

Reply:
these show up on the pypi page, but they're purely a nice-to-have. removed.